native/cocos/renderer/pipeline/Define.h

```diff
@@ -397,6 +397,7 @@ struct CC_DLL UBOSH {
 };
 
 enum class CC_DLL ForwardStagePriority {
+    AR      = 5,
```

Comment:
How is this enum used?

Reply:
this enum is for ar stage, rendering ar background before forward stage

Reply:
I see. Custom pipeline will not use this ForwardStagePriority in the future. The type of the corresponding `stage` field in effect asset will be changed to `string` instead of `enum`.

Reply:
I think we should use the enum in this version. And if engine changes the usage in the future, we will follow the new way.